dependencies.dot

```diff
@@ -1,14 +1,16 @@
+# this dotfile is used as a reference source for project dependencies
+# each folder entry must have a "label" equal to its directory name
 digraph {
-  Pace [shape=box]
-  fv3core [shape=oval]
-  driver [shape=oval]
-  physics [shape=oval]
-  util [shape=oval]
+  pace [shape=box]
+  fv3core [shape=oval, label="fv3core"]
+  driver [shape=oval, label="driver"]
```

Comment:
driver does not exist yet. should it already be here?

Reply:
No harm in including it because it's not depended on by any other components, it can happen now or when the folder gets added. I believe it will be added as part of Rhea's current PR(s).

Reply:
it's fine if it is here. Right now it exits not very gracefully if you call `changed_from_main.py driver` but this is very temporary. If it is not blocking any CI related work this is totally fine!